phpBB/phpbb/db/migration/data/v310/profilefield_aol.php

```diff
@@ -18,7 +18,7 @@ class profilefield_aol extends \phpbb\db\migration\profilefield_base_migration
 	static public function depends_on()
 	{
 		return array(
-			'\phpbb\db\migration\data\v310\profilefield_yahoo_cleanup',
+			'\phpbb\db\migration\data\v310\profilefield_website_cleanup',
```

Comment:
This file should be untouched.

Reply:
Yeah, but im not sure why are we depending on this one instead of the website cleanup. If yahoo gets removed then we will have to update this file as well. At the moment ill add this changes.

Reply:
No, just keep all existent files untouched and only make changes in new migration.

Reply:
ok, done.